README.md

```diff
@@ -0,0 +1,64 @@
+README
+# locales-check
+
+`locales-check` is a tiny library designed to identify discrepancies between your locale files by detecting missing keys. It analyzes the combined structure of all your locale files and provides a report on any absent keys. Currently, the library supports only JSON files, which must be stored in the same locales directory.
+
+## Content
+- [Why?](#why)
+- [Usage](#usage)
+- [Configuration](#configuration)
+- [License](#license)
+
+## Why?
+When we incorporated multiple languages into our projects, managing consistent keys across all language files became a challenge. Often, some keys were overlooked, and even code reviews occasionally failed to catch these omissions. To address this issue, we implemented this small library to then use it as a check in our Continuous Integration (CI) workflow to identify any missing keys in our locale files.
+
+
+## Usage 
+
+Install the package in your project:
+
+With npm
+
+```sh
+npm install --save-dev locales-check
+```
+With yarn
+```sh
+yarn add -D locales-check
+```
+
+
+Use it via script in your `package.json` file:
+
+```json
+{
+  "scripts": {
+    "locales-check": "locales-check --localesPath='./path/to/locales-files/'"
+  }
+}
+```
+
+## Configuration
+### `localesPath`
+
+* Name: `localesPath`
+* CLI argument: `--localesPath`
+* Required: Yes
+* Type: `string`
+* Description: The path where your locales files are placed
+* Examples: `./path/to/locales-files`
+
+### `ci`
+
+* Name: `ci`
+* CLI argument: `--ci`
+* Required: No
+* Default: `false`
+* Type: `boolean`
+* Description: Causes the process to exit with exit code 1 if at least one translation key is missing or unused (useful if it is part of a CI pipeline).
+
+
+
+## License
+
+[MIT](http://opensource.org/licenses/MIT)
```

Comment:
[SUG] I have seen a copy of the license inside some repos
```
MIT License

Copyright (c) 2024 MOK labs

Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the "Software"), to deal
in the Software without restriction, including without limitation the rights
to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
copies of the Software, and to permit persons to whom the Software is
furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
SOFTWARE.
```

Reply:
It already has a MIT  LICENSE file in the initial commit 😉 , but I will add the link in the README

Reply:
Sorry! I didn't see that the file already exists